protocol/localstatequery/README.md

```
@@ -0,0 +1,162 @@
+
+## Protocol Identifiers
+
+| Property | Value |
+|----------|-------|
+| Protocol Name | `local-state-query` |
+| Protocol ID | `7` |
+| Mode | Node-to-Client |
+
+## State Machine
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)